build/config/webpack.base.snippets.dev.js

```diff
@@ -7,7 +7,9 @@ export default {
   output: {
     pathinfo: true,
   },
-  devtool: 'eval',
+  // Something in `fathom-web/utils.js` breaks when eval'd,
+  // so comment out 'eval' tool for now???
+  // devtool: 'eval',
```

Comment:
That works. I tried some others and it didn't, can't remember which one it was though